src/SFML/Window/Android/WindowImplAndroid.cpp

```diff
@@ -413,28 +413,46 @@ int WindowImplAndroid::processKeyEvent(AInputEvent* inputEvent, ActivityStates&
     const std::int32_t key     = AKeyEvent_getKeyCode(inputEvent);
     const std::int32_t metakey = AKeyEvent_getMetaState(inputEvent);
 
-    Event::KeyChanged keyChanged;
-    keyChanged.code    = androidKeyToSF(key);
-    keyChanged.alt     = metakey & AMETA_ALT_ON;
-    keyChanged.control = false;
-    keyChanged.shift   = metakey & AMETA_SHIFT_ON;
-
     switch (action)
     {
         case AKEY_EVENT_ACTION_DOWN:
-            forwardEvent(Event::KeyPressed{keyChanged});
+        {
```

Comment:
Consider:

```cpp
const auto forwardKeyEvent = [&](auto keyEvent)
{
    keyEvent.code  = androidKeyToSF(key);
    keyEvent.alt   = metakey & AMETA_ALT_ON;
    keyEvent.shift = metakey & AMETA_SHIFT_ON;
    forwardEvent(keyEvent);
};
```

Reply:
Thank you for the tip. I implemented it.